### ISSUE TITLE
good first issue

### ISSUE THREAD

**Issue by zapalagrzegorz** [karinazz/bestofbreed#1]
@karinazz  czy mi się tylko zdaje, że pliki nie mają rozszerzeń?

**Comment by karinazz** [comment]
Tak, tutaj nie ma. Pospieszylam się i dodałam. Stronke dziś  przebudowalam inaczej takze będę chciała dodać na nowo.

**Comment by zapalagrzegorz** [comment]
@karinazz ok, czekam na zmiany i zamykam wtedy. Jak poprawisz, to możesz uruchamiać github pages w ustawieniach.
